base/linalg/arnoldi.jl

```diff
@@ -355,6 +355,7 @@ iterations derived from [`eigs`](@ref).
 **Outputs**
 
 * `svd`: An `SVD` object containing the left singular vectors, the requested values, and the right singular vectors. If `ritzvec = false`, the left and right singular vectors will be empty.
+  U, S, V and Vt can be obtained from the SVD object with `Z[:U], Z[:S], Z[:V] and Z[:Vt]` (where `Z = svds(A)[1]`), such that `A = U*diagm(S)*Vt`. The algorithm produces Vt and hence Vt is more efficient to extract than V.
```

Comment:
> The algorithm produces `Vt`

This should be something like "internally `Vt` is stored and hence...", because in contrast to LAPACK, which produces `Vt`, the algorithm used by ARPACK actually produces `V`.